libpirate/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 cmake_minimum_required(VERSION 3.13)
 project(libpirate)
 
-set(CMAKE_C_COMPILER "clang")
```

Comment:
I think the error compilation error was caused by gcc: https://github.com/GaloisInc/pirate/pull/17/checks?check_run_id=480168183.
```
/home/runner/work/pirate/pirate/libpirate/serial.c:66:20: error: comparison of unsigned expression < 0 is always false [-Werror=type-limits]
         if (wr_len < 0) {
                    ^
cc1: all warnings being treated as errors
```
Is this change to make sure that the Docker image sets cc1 to clang?

Reply:
It was made so that I could use `-DCMAKE_C_COMPILER=clang-9` (or other compilers) than just `clang`.  In principal, I guess I could use a custom path.

Reply:
```cmake -DCMAKE_C_COMPILER="clang-9" ..``` should work now. I merged a patch that cleaned up how we use CMAKE_C_COMPILER.